### DISCUSSION TITLE
while installing nginx giving the below error

### DISCUSSION THREAD

**Discussion by manoj531222** [learndevopsonline/learndevopsonline#1325] [Q&A]
hi raghu,
please do neeedful
![image](https://github.com/learndevopsonline/learndevopsonline/assets/129020030/64b7f829-8c7e-4b0f-8d3b-bcd0dac4cf90)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/129020030/a9371f83-4e05-4331-b06a-4b870a08cc3f)

**Comment by Naresh-vk-243** [comment]
It is showing there is an error in roboshop.conf, have you added all the server urls dns records to route53, it is showing your user-dev not added
